plugins/woocommerce-admin/client/marketing/overview-multichannel/IntroductionBanner/IntroductionBanner.tsx

```diff
@@ -42,13 +46,19 @@ export const IntroductionBanner = ( {
 	 * and it does not make sense to display the "Add channels" button in this introduction banner
 	 * that will do nothing upon click.
 	 *
+	 * This also depends on the `showCreateCampaignButton` above.
+	 * If "create campaign" button is not shown, then "add channels" button should not be shown either.
+	 * This is to make it look better in the UI.
+	 *
 	 * If there are registered channels and recommended channels,
 	 * the Channels card will display the  "Add channels" toggle button,
 	 * and clicking on the "Add channels" button in this introduction banner
 	 * will scroll to the button in Channels card.
 	 */
 	const showAddChannelsButton = !! (
-		dataRegistered?.length && dataRecommended?.length
+		dataRegistered?.length &&
+		dataRecommended?.length &&
+		showCreateCampaignButton
```

Comment:
This change ( and the comment ) is not necessary if you change the logic of line 115 to only depend on `showCreateCampaignButton` .

`{ ( showCreateCampaignButton || showAddChannelsButton ) && (` 

should be changed into:

`{ showCreateCampaignButton  && (`

at line 115 in the updated file.

Reply:
Yea, I thought about that, but I didn't do that because I thought it would be better to separate `showCreateCampaignButton` and `showAddChannelsButton` with their own logics.

But anyway, based on your comments, I looked into it and made a bit of changes in [`04294c4` (#38825)](https://github.com/woocommerce/woocommerce/pull/38825/commits/04294c4ec8301ab424bff723b54a8f770cf2863a). I changed `showCreateCampaignButton` to `showButtons`, and the code is now simplified a bit.